scripts/hurricane/stats/exevs_hurricane_regional_tropcyc_stats.sh

```diff
@@ -130,9 +130,33 @@ sed -i "s|$SEARCHx|$Model_List|g" TCPairs_template.conf
 
 run_metplus.py -c $STORMdata/TCPairs_template.conf
 
+<<<<<<< HEAD
```

Comment:
@OliviaOstwald-NOAA  -- take a closer look at this section, from `<<<<<<` to `>>>>>>`. Somehow a conflict got committed and pushed and was never resovled.